### DISCUSSION TITLE
Wyze Home Assistant Integration

### DISCUSSION THREAD

**Discussion by jkmozzz** [SecKatie/ha-wyzeapi#270] [Q&A]
I followed your install, but no "Wyze Home Assistant Integration" is available. Can you help?

Thank you,
John

**Comment by JoeSchubert** [comment]
You installed through HACS then went to integrations and there is no Wyze Home Assistant option when installing through the normal home assistant add integration dialog?

**Reply by jkmozzz** [reply]
I uninstalled, then reinstalled and finally found the wyze integration. So everything seems to be working fine. Thank you for checking back.
Would you have a suggestion for the best way to include live views of my wyze cameras?

**Reply by JoeSchubert** [reply]
There's really no great way to do this currently. You can manage it if you flash them with the rtsp firmware, there's a number of integrations that suppprr them then.

Unfortunately, nobody has cracked the webrtx handshake that they are doing as far as I know. I personally don't know enough about the protocol to crack it... I've MITM'd the initial handshke, but no idea where to go from there.

**Comment by LordNex** [comment]
Personally I’ve had to rstp the firmware and then bring them in through WebRTC to get a usable stream. Even motionEye wasn’t cutting it. But that leaves out my Outdoor Cam or most of their other products.

 I’m moving away from the Wyze line all together. With the amount of ESPHome, Aqara, ReoLink, and now Tuya devices out there that ARE open API, that it makes Wyze products now a moot point. If they would have just worked with HA like they did with IFTTT and allow a little access, my whole house would be Wyze filled. I still can’t even get my lock to work right with their app. Every other day it wants to recalibrate, so no it’s all ZHA (Works beautifully) and I only use the app to look at my soon to be replaced WOC

LoRd_NeX

On Nov 3, 2021, at 8:51 PM, Joe Schubert ***@***.***> wrote:

﻿

There's really no great way to do this currently. You can manage it if you flash them with the rtsp firmware, there's a number of integrations that suppprr them then.

Unfortunately, nobody has cracked the webrtx handshake that they are doing as far as I know. I personally don't know enough about the protocol to crack it... I've MITM'd the initial handshke, but no idea where to go from there.

**Reply by danielzrob** [reply]
I'm still holding out.. but man is it getting hard.

**Comment by jkmozzz** [comment]
Thank you.

John Mozdzierz

**Comment by jkmozzz** [comment]
Thank you.

John Mozdzierz

**Reply by michaelrobgrove** [reply]
I use Wyzs Docker Bridge to bring the cams into HA without the rtsp firmware. It brings up a rtsp and hls link
